ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -251,7 +253,7 @@ open class TwoLineTitleView: UIView {
         guard let themeView = notification.object as? UIView, self.isDescendant(of: themeView) else {
```

Comment:
Is this the `themeDidChange` function? If so, I think it can be removed since you already have `registerOnUpdate`